client/src/redux/reducers/positionReducer.js

```diff
@@ -1,22 +1,28 @@
-import { GET_COMMON_POSITIONS, SET_CURRENT_POSITION } from '../types';
+import { CURRENT_POSITION, GET_COMMON_POSITIONS, SWITCH_TO_COMMON } from '../types';
 
 const initialState = {
+  currentPositions: null,
   commonPositions: null,
-  currentPosition: null,
 };
 
 const positionReducer = (state = initialState, action) => {
   switch (action.type) {
     case GET_COMMON_POSITIONS:
```

Comment:
I wouldn't consolidate setting `currentPositions` in the same reducer. Rather, just make two dispatches.